src/Commands/Setup/ContentReplacer.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace CodeIgniter\Shield\Commands\Setup;
+
+class ContentReplacer
+{
+    /**
+     * @param array $replaces [search => replace]
+     */
+    public function replace(string $content, array $replaces): string
```

Comment:
Not sure it matters, but you could probably replace this entire method with a single `strtr()` call.

Reply:
Yes, also with a single `str_replace()`.
They are a bit differnt, but `strtr()` is fine for this case.
https://www.php.net/manual/ja/function.strtr.php#117169

Done.

Reply:
The further we go, the more I enjoy your teamwork. this is fabulous.